### PR TITLE
List Galilel (GALI)

### PR DESCRIPTION
**Pull request by mbroemme**
* Official project URL: https://galilel.cloud
* Official block explorer URL: https://explorer.galilel.cloud

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/Galilel.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Base58BitcoinAddressValidator;
+import bisq.asset.Coin;
+import bisq.asset.NetworkParametersAdapter;
+
+public class Galilel extends Coin {
+    public Galilel() {
+        super("Galilel", "GALI", new Base58BitcoinAddressValidator(new GalilelMainNetParams()), Network.MAINNET);
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Galilel.java]: Remove last param as it is default value.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Comment by mbroemme** [comment]
> NACK per comments

@blabno I've updated the pull request

**Comment by blabno** [comment]
Good, now please squash your commits into a single one.

**Comment by mbroemme** [comment]
@blabno squashed into single commit, hope all is good now.

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**